monasca_rsyslog/client.py

```diff
@@ -26,13 +29,23 @@
 cfg.CONF.register_opts([
     cfg.StrOpt('url',
                default=None,
-               help='Specify URL for the logging API')
+               help='Specify URL for the logging API'),
+    cfg.IntOpt('max_batch_size',
+               default=100,
+               min=1,
+               help='Specify maximum buffer size until triggering a post'),
+    cfg.IntOpt('min_poll_delay',
+               default=10,
+               help='Specify minimum time to wait between triggering a post'),
+    cfg.BoolOpt('verbose',
```

Comment:
nit: Maybe this should live outside the API section in the conf file

Reply:
We can deal with this when we infer the api URL via keystone.